<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RLCUH5K" failures="0" tests="3" name="org.test.B" time="41.661" errors="1" timestamp="19 Apr 2024 11:26:23 GMT" skipped="0">
  <testcase classname="org.test.B" name="tc27" time="14.622"/>
  <testcase classname="org.test.B" name="tc25" time="27.029"/>
  <testcase classname="org.test.B" name="tc22" time="0.010">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.baseclass.BaseClass.launchUrl(BaseClass.java:50)
at org.test.B.tc22(B.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
]]>
    </error>
  </testcase> <!-- tc22 -->
</testsuite> <!-- org.test.B -->
